Conceptpower+Spring/src/main/java/edu/asu/conceptpower/core/ReviewRequest.java

```diff
@@ -0,0 +1,35 @@
+package edu.asu.conceptpower.core;
+
+
+public class ReviewRequest {
+
+
+    private String conceptId;
```

Comment:
we still need requester and resolver. we want to know who made a request and who addressed it.